### ISSUE TITLE
:poss and its reification

### ISSUE THREAD

**Issue by jheinecke** [amrisi/amr-guidelines#261]
The documentation says about `:poss` 

>The relation :poss (“possessed by”) is a very general form of possession. AMR uses it only for possessives and prepositional >phrases with “of”:
> ```
> (c / car
>   :poss (h / he))
> ```
so here _c_ (domain of the `:poss` relation)  is the owned item (corresponding to `:ARG1` of `own-01` or `have-03` )
and  _h_ (range of `:poss`) is the possessor (corresponding to `:ARG0` of `own-01`  or `have-03` ).
However in the reification table this is shown inverted:

Relation | Reification | Domain | Range
-- | -- | -- | -- 
:poss | own-01, have-03 | :ARG0 | :ARG1

Is this an error in the reification table ?

**Comment by nschneid** [comment]
Yes I think you're right—`:poss` is an unusual relation in that, when reified with a predicate of ownership, the concept it introduces has a lower-numbered ARG than the concept it modifies.

**Comment by jheinecke** [comment]
Thanks! I'll modify the table and create a PR
